en/deploy/README.md

```diff
@@ -47,7 +47,7 @@ db.sqlite3
 
 And save it as `.gitignore` in the "djangogirls" folder.
 
-> **Note** The dot at the beginning of the file name is important!  If you're having any difficulty creating it (Macs don't like you to create files that begin with a dot via the Finder, for example), then use the "Save As" feature in your editor; it's bulletproof.
+> **Note** The dot at the beginning of the file name is important!  If you're having any difficulty creating it (Macs don't like you to create files that begin with a dot via the Finder, for example), then use the "Save As" feature in your editor; it's bulletproof. And be sure not to add .txt, .py, or any other extension to the file name -- it will only be recognized by Git if the name is just `.gitignore`.
```

Comment:
Can you mark up the extensions as inline code, by wrapping them in <code>`</code>s?

I'm not a native speaker of English, but I think you can add further emphasis to the phrasing by swapping "not" and "to":
> And be sure to not add `.txt`, `.py`, or any other extension to the file name -- it will only be recognized by Git if the name is just `.gitignore`.

Reply:
I code escaped the extensions.

I am a native speaker of (American) English... To me, "Be sure to not add" sounds very odd, while "Be sure not to add" is more normal. I agree it might be more emphasis, but it really sounds awkward to a native speaker (at least to me).